simpleapp/src/main/res/layout/activity_dynamic_photos.xml

```diff
@@ -14,11 +14,10 @@
         <com.pixlee.pixleesdk.ui.widgets.list.PXLWidgetView
```

Comment:
Is it possible to add an attribute here to specify which type of widget will be used or does that always have to happen in the Java/Kotlin?

Reply:
Yes, it is possible. As long as the attributes are intuitively easy to use, they can be good options.